app/models/manageiq/providers/openstack/manager_mixin.rb

```diff
@@ -141,7 +141,7 @@ def translate_exception(err)
   end
 
   def verify_api_credentials(options = {})
-    options[:service] = "Compute"
+    options[:service] = "Identity"
```

Comment:
The compute service was needed since multiregion support. Multiregion OpenStack deployments can share Identity service, but can have multiple Compute (or other) services in various regions. So it make sense to test region-scoped Compute service instead of shared Indetity service.

Could the PR be updated to use Compute service with a default tenant or some different technique?